controllers/front/CartController.php

```diff
@@ -63,6 +68,7 @@ public function init()
         $this->customization_id = (int)Tools::getValue('id_customization');
         $this->qty = abs(Tools::getValue('qty', 1));
         $this->id_address_delivery = (int)Tools::getValue('id_address_delivery');
+        $this->preview = Tools::getValue('preview', 0);
```

Comment:
This is not a boolean. It would be better to test for the exact value that is true instead:
```php
$this->preview = ('1' === Tools::getValue('preview'));
```